.github/workflows/build-lint-test.yml

```diff
@@ -148,13 +150,15 @@ jobs:
           path: |
             packages/snaps-execution-environments/dist/browserify
           key: snaps-execution-environments-build-${{ runner.os }}-18.x-${{ github.sha }}
+          fail-on-cache-miss: true
       - name: Cache Webpack vendor
         id: cache-webpack-vendor
         uses: actions/cache@v3
         with:
           path: |
             packages/snaps-simulator/vendor
           key: webpack-vendor-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+          fail-on-cache-miss: true
```

Comment:
This one can fail validly right?

Reply:
Good catch, it should be able to fail there.